src/Custard/ScenarioDescription.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rhubarb\Stem\Custard;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ScenarioDescription
+{
+    private $lines = [];
+
+    public function writeLine(string $line)
+    {
+        $this->lines[] = "   " . $line;
```

Comment:
Potentially we could have an optional $spacer variable here defaulting to "   " instead of the hard implementation?